src/plugins/kibana_overview/public/components/add_data/add_data.test.tsx

```diff
@@ -33,7 +32,7 @@ const mockFeatures = [
     title: 'Add Elastic Agent',
   },
   {
-    category: FeatureCatalogueCategory.DATA,
+    category: 'data' as const,
```

Comment:
I believe TS 4+ automatically infers the `as const` part so no need to explicitly state it. Was TS complaining?

Reply:
Yea, it was :/